README.md

```diff
@@ -40,6 +54,10 @@ See our [list of frequently-asked questions](./etc/FAQ.md).
 - As of today, `coq-lsp` may have trouble when more than one file is open at
   the same time, this is a problem upstream. For now, you are advised to
   work on a single file if this problem appears.
+- If you install `coq-lsp` simultaneously with VSCoq, VSCode gets confused and
+  neither of them will work. Help with resolving this
```

Comment:
Supposedly this will be resolved with #210

Reply:
Yes, still worth mentioning. #210 doesn't solve the issue, just warns the user about it. A extension can't disable another TTBOMK. Of course we could in the future coordinate with VSCoq to allow both extensions to be installed, but that's a different story.